src/wkb2gltf.core.tests/wkb2gltf.tests.csproj

```diff
@@ -19,6 +19,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageReference Include="SharpGLTF.Toolkit" Version="1.0.0" />
```

Comment:
this is not needed, reference to SharpGltf is defined in project wkb2gltf.core

Reply:
Removed it